src/components/OSCALControlGuidance.test.js

```diff
@@ -5,7 +5,9 @@ import OSCALControlGuidance from "./OSCALControlGuidance";
 
 test("OSCALCatalog displays control guidance", async () => {
```

Comment:
Since we are editing the test, I do not think that the description is entirely accurate about what is actually being tested. `OSCALCatalog` is not even rendered by this test.